### ISSUE TITLE
Dashboard widgets is blocked by an invisible element

### ISSUE THREAD

**Issue by rizkysyazuli** [yogiben/meteor-admin#172]
I'm just starting out when I noticed there's an invisible `#admin-delete-modal` element blocking the dashboard. Any ideas what might have caused this? Didn't saw this in your [meteor-starter](https://github.com/yogiben/meteor-starter) demo.

![screenshot 2015-03-23 18 40 00](https://cloud.githubusercontent.com/assets/152221/6796581/40a55516-d228-11e4-90bd-5f52d698c1cb.png)

**Comment by mpowaga** [comment]
@rizkysyazuli Can you provide more details (browser etc.)?

**Comment by rizkysyazuli** [comment]
@mpowaga hey, i think it's caused by overlapping styles from the other packages. i've created another issue for this in #176 

so let me just close this one.
